### ISSUE TITLE
angular-clockpicker not working 

### ISSUE THREAD

**Issue by cvssandhu** [xjpro/angular-clockpicker#1]
I have included files and try to make it work but something missing could you please upload working example with html code.

Its not showing any error I have added module in my project.

**Comment by xjpro** [comment]
Hi. Would you mind posting some of your code? I might be able to help out.

**Comment by cvssandhu** [comment]
Hi,Thanks for support in advance..

Please find my code as in attachment.

Thanks,
Chakraveer Singh Sandhu

**Comment by cvssandhu** [comment]
please find the codepen for this i have added timepicker js under js section..

http://codepen.io/cvssandhu/pen/vOQVjo

**Comment by xjpro** [comment]
Hi @cvssandhu. Thanks for including the codepen. It made it easier to see that you were doing everything correctly according to the README. The issue is that I had since changed how the code worked a bit and didn't update the README to reflect those changes. I've now done that. Usage was changed to be:

``` html
<clockpicker ng-model="time"></clockpicker>
```

As part of this I also updated the CSS so it looks like it was not aligning quite right.
